index.html

```diff
@@ -229,9 +229,13 @@
             <li class="list-inline-item"><a href="http://ng-table.com/">ngTable</a></li>
         </ul>
     </div>
-
     <script src="https://cdn.jsdelivr.net/npm/angular@1.7.8/angular.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/ng-table@3.0.1/bundles/ng-table.min.js"></script>
     <script src="./script.js"></script>
+    
+    <div class="btn-group btn-group-sm scroll-btns-grp" role="group" aria-label="Scroll buttons">
+      <button type="button" onclick=topFunction() title="Go to the top" class="btn page-link">▲</a>
```

Comment:
We should wrap the `onclick` with double quotes. E.g., `onclick="topFunction()"`.

And personally, I don't use the `role` and `aria-*` attributes. Should be safe to remove them.

Reply:
Done.